### PR TITLE
Deregister features from fortran backend which are not in use yet (for which no tests exist)

### PR DESCRIPTION
**Pull request by HarshitaKalani**
I've also shifted the intrinsic Inquiry functions from elemental functions.
Towards: https://github.com/lfortran/lfortran/pull/4054#discussion_r1611988947, #4017

### CODE REVIEW

File: src/libasr/codegen/asr_to_fortran.cpp [hunk 1]
```diff
@@ -1374,35 +1360,7 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
             SET_INTRINSIC_NAME(Dim, "dim");
             SET_INTRINSIC_NAME(Ifix, "ifix");
             SET_INTRINSIC_NAME(Idint, "idint");
-            SET_INTRINSIC_NAME(Epsilon, "epsilon");
-            SET_INTRINSIC_NAME(Precision, "precision");
-            SET_INTRINSIC_NAME(Tiny, "tiny");
```
Comment (certik) [inline src/libasr/codegen/asr_to_fortran.cpp:1379]: The above 3 are used by Fortran.

File: src/libasr/codegen/asr_to_fortran.cpp [hunk 2]
```diff
@@ -1374,35 +1360,7 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
...
-            SET_INTRINSIC_NAME(BitSize, "bit_size");
-            SET_INTRINSIC_NAME(NewLine, "new_line");
             SET_INTRINSIC_NAME(Conjg, "conjg");
-            SET_INTRINSIC_NAME(Huge, "huge");
```
Comment (certik) [inline src/libasr/codegen/asr_to_fortran.cpp:1383]: Huge is used by Fortran.

File: src/libasr/codegen/asr_to_fortran.cpp (hunk 1 above)
Comment (HarshitaKalani) [reply src/libasr/codegen/asr_to_fortran.cpp:1379]: Yes, these are registered in the visit_TypeInquiry() (where they belong to), so I removed them from visit_IntrinsicElementalFunction().

File: src/libasr/codegen/asr_to_fortran.cpp (hunk 2 above)
Comment (HarshitaKalani) [reply src/libasr/codegen/asr_to_fortran.cpp:1383]: Same for Huge.

File: integration_tests/CMakeLists.txt [hunk 3]
```diff
@@ -542,7 +543,7 @@ RUN(NAME int_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm) # int body
 RUN(NAME int_02 LABELS gfortran wasm) # int symboltable
 RUN(NAME int_03 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc) # large int and prints array
 
-RUN(NAME intrinsics_01 LABELS gfortran) # sqrt, abs, log
+RUN(NAME intrinsics_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc fortran) # sqrt, abs, log
```
Comment (Shaikh-Ubaid) [inline integration_tests/CMakeLists.txt:546]: Good that we enabled all the supporting backends.

File: src/libasr/codegen/asr_to_fortran.cpp [hunk 4]
```diff
@@ -1346,26 +1344,14 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
             SET_INTRINSIC_NAME(Spacing, "spacing");
             SET_INTRINSIC_NAME(BesselJN, "bessel_jn");
             SET_INTRINSIC_NAME(BesselYN, "bessel_yn");
-            SET_INTRINSIC_NAME(Mvbits, "mvbits");
```
Comment (Shaikh-Ubaid) [inline src/libasr/codegen/asr_to_fortran.cpp:1349]: `mvbits` is a fortran intrinsic, I believe. But we don't test it at the moment, I guess.

**Review by Shaikh-Ubaid — APPROVED** [review]
Good PR. LGTM. Thanks for this!

File: src/libasr/codegen/asr_to_fortran.cpp (hunk 4 above)
Comment (HarshitaKalani) [reply src/libasr/codegen/asr_to_fortran.cpp:1349]: Yes true. The file containing use of `mvbits` generates wrong fortran code and hence fails. 
I've mentioned that here: https://github.com/lfortran/lfortran/issues/3416#issuecomment-2128524919

**Pull request merged**